components/Blog/styling/Blog.styles.ts

```diff
@@ -0,0 +1,61 @@
+import styled from 'styled-components';
+
+import {
+  LargeParagraph,
+  H2,
+  H3,
+  DefaultParagraph,
+} from 'components/atoms/typography.styles';
+
+export const TopDescription = styled(LargeParagraph)`
+  padding-bottom: 50px !important;
+  font-style: italic;
+
+  @media screen and (min-width: 768px) {
+    font-size: 36px;
+    line-height: 120%;
+    font-weight: 400;
+    padding-bottom: 55px !important;
+  }
+`;
+
+export const MainTitle = styled(H2)`
+  padding-bottom: 20px !important;
+
+  @media screen and (min-width: 768px) {
+    font-size: 48px;
+    line-height: 57.6px;
+    padding-bottom: 40px !important;
+  }
+`;
+
+export const MainContainer = styled.div`
```

Comment:
While I've been looking at the code, I've noticed that there are 4-5 MainContainer components. I had a hard time looking for a specific one. Feel free to use longer and more descriptive names.

Reply:
A huge part of the reason I keep it this lazy is because the complete opposite of this happened at one of the first PRs. Since it's closed I cannot find it, but I know it was Imre who did not like it that way.

As I see it, the pros for long descriptive names:
- makes the clone cleaner for sure

And the cons:
- if you look at the code in the Dev Tools Inspect, the compiled class names tell you where to find it. E.g. in this case this one is 'Blogstyles__MainContainer-sc-kpxgsh-2 jTgBly', (the first part of) which directs you to the file you should look for it. I am not sure, but I think Imre's argument was something similar the other time.

Reply:
Oh and to be fair, the other huge reason I act so lazy here is because I AM lazy 😄